chains/lisk/types/account.go

```diff
@@ -0,0 +1,47 @@
+package types
+
+type Account struct {
```

Comment:
Why do you need all these JSON struct? You should be able to create a transaction from generated go files from protobuf, create a transaction from protobuf and then send the tx to Lisk chain